kuma/wiki/models.py

```diff
@@ -920,6 +923,9 @@ def save(self, *args, **kwargs):
             # If this is a translation without a topic parent, try to get one.
             self.acquire_translated_topic_parent()
 
+        # Set the shortened URL (e.g. bit.ly).
+        self.set_share_url(save=False)
```

Comment:
We have 2 places where we're calling this. On save (here) and on document display if it's not set already. Would we want tasks for both?

On save it seems like we aren't too concerned with speed but we're also not concerned with a delay in getting the bitly URL.

On display we don't want to block but also what do we do assuming the task doesn't get the bitly URL in time... I could trigger the task and return the full URL. The next time it is loaded the document would have the bitly URL.